react_agent/api.py

```diff
@@ -0,0 +1,56 @@
+from dotenv import load_dotenv
```

Comment:
May be optional but, i have to add this line in order to prevent `Module not found error` for react_agent. In order to ensure the root directory is included in Python's module search path.

```
import os
import sys

sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Reply:
oh interesting, let me check this

Reply:
I got the same error as well, included this line 👍